projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts

```diff
@@ -27,9 +27,8 @@ export class DataLayerColorService {
     line.backgroundColor = color + '33'; // temporary dirty hack to set opacity. assumes color is in 6-character hex format.
   }
 
-  getAnnotationColor(annotation: any): string | undefined {
-    const color = annotation.backgroundColor;
-    if (typeof color === 'string' && !color.includes('33')) {
+  addTransparencyAbritraryColor(color: any): string | undefined {
```

Comment:
Can we shorten this method name to `addTransparency`?

Also, please specify a type for the parameter.

Reply:
Okay, I am changing this method name from addTransparencyAbritraryColor to addTransparency.